Automattic/jetpack
# Add/explat components

Context: pbNhbs-aMJ-p2

## Proposed changes:

* This PR implements a package to use the React interface for ExPlat, sending the information through the Jetpack API:
  * The React interface relies on [@automattic/explat-client-react-helpers](https://github.com/Automattic/wp-calypso/blob/trunk/packages/explat-client-react-helpers/README.md) and [@automattic/explat-client](https://github.com/Automattic/wp-calypso/tree/trunk/packages/explat-client)
  * The package also registers a backend route that is used to fetch [the current assignment](https://github.com/Automattic/jetpack/blob/add/explat-components/projects/packages/explat/src/client/assignment.ts#L16)

For more information about ExPlat, go to PCYsg-tdK-p2

### Other information:


## Jetpack product discussion


pbNhbs-aMJ-p2

## Does this pull request change what data or activity we track or use?


Yes. This PR implements a way to run A/B tests in the plugin using ExPlat, similar to how we run experiments in Calypso and WooCommerce.

## Testing instructions:






* Checkout this branch locally
* Confirm that it's building correctly (when running `jetpack build packages/explat` and `jetpack watch packages/explat`)
* To test the functionality, follow the testing instructions in #37973
